kinder/ci/workflows/presubmit-upgrade-beta-master.yaml

```diff
@@ -0,0 +1,135 @@
+version: 1
+summary: |
+  this workflow tests kubeadm upgrades from Kubernetes version ci/k8s-beta (the latest unstable release) to version ci/latest
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#pull-kubeadm-kinder-upgrade-beta-latest
+  config    > https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/kubeadm/kubeadm-presubmits.yaml
+vars:
+  initVersion: "{{ resolve `ci/k8s-beta` }}"
+  upgradeVersion: "{{ resolve `ci/latest` }}"
```

Comment:
I'm not fully aware of the differences between `latest` label series and the `k8s` label series, but I find a little bit weird to mix the two. Then, considering we are using `k8s-beta`, I guess this should be `k8s-master`

Reply:
k8s-beta can give a different version from latest
k8s-master is always the same as latest.

i will do `ci/latest` for both, but the upgrade will be a NO-OP.